Source/DafnyCore/Options/ICommandSpec.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.IO;
+
+namespace Microsoft.Dafny;
+
+public interface ICommandSpec {
+
+  public static Argument<FileInfo> FileArgument { get; }
+
+  private static ValidateSymbolResult<ArgumentResult> ValidateFileArgument() {
+    return r => {
+      var value = r.Tokens[0].Value;
+      if (value.StartsWith("--")) {
+        r.ErrorMessage = $"{value} is not a valid argument";
```

Comment:
Why?

Reply:
Removed it. I think it was an ill-conceived attempt at warning the user against accidentally passing options as files.